resources/schemas/representation-1.1.0.yaml

```diff
@@ -0,0 +1,209 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://astropy.org/schemas/astropy/coordinates/representation-1.1.0"
+
+title: |
+  Representation of points or differentials in two or three dimensional space.
+
+description: |
+  Representation of points or differentials in two or three dimensional space.
+
+examples:
+  -
+    - A SphericalRepresentation
+    - |
+        !<tag:astropy.org:astropy/coordinates/representation-1.1.0>
+          components:
+            distance: !unit/quantity-1.1.0 {unit: !unit/unit-1.0.0 AU, value: 1.0}
+            lat: !<tag:astropy.org:astropy/coordinates/latitude-1.0.0> {unit: !unit/unit-1.0.0 deg,
+              value: 10.0}
+            lon: !<tag:astropy.org:astropy/coordinates/longitude-1.0.0>
+              unit: !unit/unit-1.0.0 deg
+              value: 10.0
+              wrap_angle: !<tag:astropy.org:astropy/coordinates/angle-1.0.0> {unit: !unit/unit-1.0.0 deg,
+                value: 360.0}
+          type: SphericalRepresentation
+  -
+    - A CartesianDifferential
+    - |
+        !<tag:astropy.org:astropy/coordinates/representation-1.1.0>
+          components:
+            d_x: !unit/quantity-1.1.0 {unit: !unit/unit-1.0.0 km s-1, value: 100.0}
+            d_y: !unit/quantity-1.1.0 {unit: !unit/unit-1.0.0 km s-1, value: 200.0}
+            d_z: !unit/quantity-1.1.0 {unit: !unit/unit-1.0.0 km s-1, value: 3141.0}
+          type: CartesianDifferential
```

Comment:
asdf-astropy does not currently support the 1.1.0 tag meaning this will fall back on astropy which fails due to warnings.